### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by JCliffardo**
Added the Meta Pixel to the bottom of the header

Added a button link under the contact section for the enquiry web form.

The pixel I'm comfortable is right, the button link not so sure of. I want the format to fit the look of the page so my knowledge is limited to basic functions, a rounded button would be nice if it's not and the same hex colour would also be nice.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -274,6 +291,9 @@ <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
         <h2 class="font-vag-rounded text-3xl text-geblue dark:text-white">
           Contact
         </h2>
+        <a href="https://tradehq.com/groundedelectricalservices/enquire">
+          <button>Request a Quote</button>
+        </a>
```
Comment (seanclifford) [inline src/index.html]: I'll check if this looks OK tonight. It might work just like this, but I'm unsure - it'll probably need some css styling

**Comment by seanclifford** [comment]
@JCliffardo - see what you think. I made the button more blue to stand out a bit more.

**Comment by JCliffardo** [comment]
Button looks great! Might have to play around with the text. Maybe all caps, maybe just bold, maybe both. I can play with that though. Thanks heaps. 

Is it at all possible though to have the button appear different colours depending on dark mode / light mode? It washes into the background on dark. 

Kind regards,
Joe

**Comment by seanclifford** [comment]
Yes - if you have a colour scheme in mind for the dark mode I can put that in.

**Pull request merged**